src/main/java/de/hhu/propra/sharingplatform/dto/UserForm.java

```diff
@@ -1,4 +1,4 @@
-package de.hhu.propra.sharingplatform.form;
+package de.hhu.propra.sharingplatform.dto;
```

Comment:
This form will be deleted after refactor by @ik8472

Reply:
This change occurs just cause we changed the pacckage name of the "form" package because we added two more classes. So this can be ignored and refactored at any time. The new changes are completely independent from the class.